backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/QueueArguments.java

```diff
@@ -31,7 +31,7 @@ public Builder deadLetter(String deadLetterQueueName) {
         }
 
         public Builder queueTTL(long queueTTL) {
-            arguments.put("x-expires", queueTTL);
+            arguments.put("x-message-ttl", queueTTL);
```

Comment:
No it is not the same thing...

Reply:
I think you misunderstood here. We want to put a TTL on the queue, not on the messages in the queue :) Thus your added test at the end does not make sense as well in the context that we want

Reply:
you are right, I re-read the ticket, I  misunderstood here